custom-namespaces.yaml

```
@@ -1183,6 +1183,13 @@ subscription_platform:
       measures:
         count:
           type: count
+    active_subscriptions:
+      type: table_view
+      tables:
+        - table: mozdata.subscription_platform.active_subscriptions
+      measures:
+        count:
+          type: sum
```

I believe `sum` measures need a `sql` parameter to specify what to sum.
```suggestion
        count:
          type: sum
          sql: ${count}
```